sass/responsive/_mobile.scss

```diff
@@ -70,9 +70,108 @@
         }
     }
 
+    #submenuModal {
```

Comment:
maybe we should move all this things to the menu_item css under a media query. or more specifically to a css for the subMenuItem component.

Reply:
done.  moved to _submenu_item.scss